apitesters/purchases.ts

```diff
@@ -75,6 +77,11 @@ async function checkPurchasing(purchases: Purchases,
     paymentDiscount
   );
 
+  const result5: MakePurchaseResult = await Purchases.purchaseSubscriptionOption(
+    subscriptionOption,
+    upgradeInfo
+  );
```

Comment:
I think we are missing a bunch of new API tests for the new parameters in the different purchase methods? Also, for this one we should have some with the `isPersonalizedPrice` parameter